src/App/App.js

```diff
@@ -27,42 +27,55 @@ class App extends Component {
           loading: false,
         });
       })
-      .catch((err) => this.setState({ error: parseInt(err.toString().split('Error: ')[1]) }));
+      .catch((err) =>
+        this.setState({ error: parseInt(err.toString().split("Error: ")[1]) })
+      );
   }
 
   changeSearch = (newValue) => {
-    this.setState({searchValue: newValue})
-  }
+    this.setState({ searchValue: newValue });
+  };
 
   clearInput = () => {
-    this.setState({searchValue: ''})
-  }
+    this.setState({ searchValue: "" });
+  };
 
   render() {
-    let filteredMovies = this.state.movies.filter(movie => movie.title.includes(this.state.searchValue))
+    let filteredMovies = this.state.movies.filter((movie) =>
+      movie.title.toLowerCase().includes(this.state.searchValue.toLowerCase())
```

Comment:
Great thinking on the toLowerCase thing. I’d noticed I had to match capitalization but all the solutions I could think of seemed too complex to be in scope (editing each entry in the data array, matching unicode for upper and lower case...other bad ideas)